### DISCUSSION TITLE
How translate kbar options?

### DISCUSSION THREAD

**Discussion by armandinhox** [refinedev/refine#6033] [Q&A]
What better method?
How translate kbar options?

**Comment by alicanerdurmaz** [comment]
Hello @armandinhox, can you give us more details, please?

<img width="300" alt="Screenshot 2024-06-10 at 16 18 38" src="https://github.com/refinedev/refine/assets/23058882/2c397b5a-b1af-4cda-a7c8-66cc56962c3a">

default options are generated from resources and they have full i18n support. You can read more about this topic [here](https://refine.dev/docs/guides-concepts/i18n/#adding-the-translations-files).
